Readme (#22)
2pinetwork/Loans: README.md

Try running some of the following tasks:

```shell
yarn test
yarn coverage
```

# Collateral Pool
Pool used to deposit assets to accomplish 2 main things:
- Be used as collateral to borrow other assets
    - Collateral ratio to be used is defined per pool/asset
- Generate yield (independently of collateral/borrow use)
    - the interest can be autocompounded in the same yield to ingress the yield
    - the interest can be use to auto-repay the debt

Details:
- Collateral ratio can be set & changed
- Can be used only with one asset (defined at deploy time)
- Deposits can be limited per user & per pool
- Withdrawal fee can be defined (max 1%)
- Can be used with or without an underlying strategy to generate yield

On each deposit it returns a share token "2pi-C-assetSymbol" and it's burned on each withdrawal.

> > Note: The yield can't be chosen to auto-repay debt and to be autocompounded at the same time it do one or the other.

Reviewer: ```suggestion
> Note: The yield can't be chosen to auto-repay debt and to be autocompounded at the same time, these functions are mutually exclusive.
```

# Liquidity Pool
Pool used to provide liquidity for borrowers. Liquidity to be borrowed is independent from Collateral pool.

Details:
- The due date of the pool is defined at deploy time.
- The interest rate and origination fee can be configured only when no-one has a debt.
- The repayment can be moved to a "safeBox" to not be borrowed again.

On each deposit it returns a share token "2pi-L-assetSymbol" and burned on each withdrawal.

### Borrowing process
Depending on the amount of collateral deposited (and its usage ratio) the user can borrow assets.

> On each borrow LiquidityPool mint debt tokens [2pi-D-assetSymbol] (for original debt & a different for interest to be paid).

Reviewer: ```suggestion
On each borrow LiquidityPool mint debt tokens [2pi-D-assetSymbol] (for original debt & a different token for the interest to be paid).
```
